### PR TITLE
Refs 269: add more arch detail to docs

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-269

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: docs/db-model.puml [hunk 1]
```diff
@@ -26,10 +32,12 @@ entity "**repository_configurations**" {
   *""name"": //character varying(255) //
   ""versions"": //character varying(255)[] //
   *""arch"": //character varying(255) //
```
Comment (Andrewgdewar) [inline docs/db-model.puml:34]: If the * is supposed to mean required, I don't think arch actually is.

File: docs/db-model.puml [hunk 2]
```diff
@@ -26,10 +32,12 @@ entity "**repository_configurations**" {
...
-  *""account_id"": //character varying(255) //
+  ""account_id"": //character varying(255) //
   *""org_id"": //character varying(255) //
   *""org_id"": //character varying(255) //
   *""repository_uuid"": //uuid [FK]//
+  ""gpg_key"": //text //
+  *""metadata_verification"": //boolean //
```
Comment (Andrewgdewar) [inline docs/db-model.puml:40]: I'm unsure if metadata_verification can be undefined as well (if the * is as suggested above).

**Review by Andrewgdewar — APPROVED** [review]
Looks good.

File: docs/db-model.puml (hunk 1 above)
Comment (jlsherrill) [reply docs/db-model.puml:34]: i believe its required in the db, but not required in the api.   In the DB its stored as empty string if not passed in  (this is auto generated puml)

File: docs/db-model.puml (hunk 2 above)
Comment (jlsherrill) [reply docs/db-model.puml:40]: same as above, its required in the db schema, but defaults to false

**Pull request merged**